.github/workflows/jobs.yml

```diff
@@ -240,17 +251,20 @@ jobs:
       - name: SCP tarball to VM
         run: scp mirror-registry.tar.gz jonathan@quay:~/mirror-registry.tar.gz
 
+      - name: Add quay to /etc/hosts
+        run: ssh jonathan@quay 'sudo echo "127.0.0.1 quay" >> /etc/hosts'
+
       - name: Install podman
         run: ssh jonathan@quay 'sudo subscription-manager refresh; sudo yum -y install podman'
 
-      - name: Log into podman for registry.redhat.io
-        run: ssh jonathan@quay "sudo podman login -u ${{ secrets.REGISTRY_USERNAME }} -p ${{ secrets.REGISTRY_PASSWORD }} registry.redhat.io"
-        if: matrix.installer-type == 'online'
-
       - name: Disable outbound network traffic
         run: ssh jonathan@quay 'sudo firewall-cmd --direct --add-rule ipv4 filter OUTPUT 0 -m state --state ESTABLISHED,RELATED -j ACCEPT; sudo firewall-cmd --direct --add-rule ipv4 filter OUTPUT 1 -p tcp -m tcp --dport=22 -j ACCEPT; sudo firewall-cmd --direct --add-rule ipv4 filter OUTPUT 1 -p tcp -m tcp --dport=8443 -j ACCEPT; sudo firewall-cmd --direct --add-rule ipv4 filter OUTPUT 2 -j DROP'
         if: matrix.installer-type == 'offline'
 
+      - name: Log into podman for registry.redhat.io
```

Comment:
Any reason that this was moved down here?

Reply:
Hmm, I don't remember, I don't think it's necessary. I'll revert.